UnityProject/Assets/Scripts/Effects/Shake.cs

```diff
@@ -35,7 +47,15 @@ public void startShake(float duration, float distance, float delayBetweenShakes)
     public void haltShake()
     {
         StopAllCoroutines();
-        transform.position = originalPosition;
+        switch (shakeType)
```

Comment:
I think codacy is in the right here, you could do the same with a simple bool and an if.